recipe/meta.yaml

```diff
@@ -11,46 +11,64 @@ source:
   sha256: 2ad7f0c8010ebbc46319cc30febfb8e59ccf84969a201541c207bc3a4fa6cf81
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
-requirements:
-  host:
-    - python >=3.8
-    - pip
-    - hatchling
-  run:
-    - packaging
-    - python >=3.8
-    - importlib-metadata
-    - typing-extensions >=4.0.1
-    - jinja2
-    - jsonschema >=3.0
-    - numpy
-    - pandas >=0.25
-    - toolz
+outputs:
+  - name: altair
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+        - hatchling
+      run:
+        - packaging
+        - python >=3.8
+        - importlib-metadata
+        - typing-extensions >=4.0.1
+        - jinja2
+        - jsonschema >=3.0
+        - numpy
+        - pandas >=0.25
+        - toolz
+    test:
+      source_files:
+        - altair
+      requires:
+        - pytest
+        - sphinx
+        - docutils
+        - vega_datasets >=0.9.0
+        - anywidget >=0.9.0
+        - recommonmark
+        - ipython
+        - pip
+      commands:
+        - pip check
+        - python -m pytest --pyargs --doctest-modules altair
+      imports:
+        - altair
+        - altair.expr
+        - altair.utils
+        - altair.vegalite
 
-test:
-  source_files:
-    - altair
-  requires:
-    - pytest
-    - sphinx
-    - docutils
-    - vega_datasets
-    - recommonmark
-    - ipython
-    - pip
-    - anywidget
-  commands:
-    - pip check
-    - python -m pytest --pyargs --doctest-modules altair
-  imports:
-    - altair
-    - altair.expr
-    - altair.utils
-    - altair.vegalite
+  - name: altair-all
+    requirements:
+      host:
+        - python >=3.8
+      run:
+        - vl-convert-python >=1.1.0
+        - pyarrow >=11
+        - vegafusion-python-embed >=1.5.0
+        - {{ pin_subpackage('altair', exact=True) }}
+        #   - altair_tiles >=0.3.0
```

Comment:
I couldn't find a way to install from `pip` in the conda recipe as it seems like the approach used in local conda envs (`pip:`) does not work. If anyone knows how to install `altair_tiles` from pip as part of this setup file please chime in (or if you feel like creating a conda package @binste).

Reply:
Thank you very much @joelostblom for this PR! I've never created a conda package before so appreciate it that you started this :)

I might have some time on the weekend to try to create a conda package as well for `altair_tiles` so that we can include it here.